apps/trading/client-pages/markets/markets-page.tsx

```diff
@@ -13,58 +9,213 @@ import {
+  } = {
+    open: {
+      id: 'open-markets',
+      name: t('Open'),
+    },
+    proposed: {
+      id: 'proposed-markets',
+      name: t('Proposed'),
+    },
+    closed: {
+      id: 'closed-markets',
+      name: t('Closed'),
+    },
+  };
 
   return (
-    <div className="h-full pt-0.5 pb-3 px-1.5">
-      <div className="h-full my-1 border rounded-sm border-default">
-        <Tabs storageKey="console-markets">
-          <Tab
-            id="open-markets"
-            name={t('Open markets')}
-            settings={<MarketsSettings />}
-          >
-            <ErrorBoundary feature="markets-open">
-              <OpenMarkets />
-            </ErrorBoundary>
-          </Tab>
-          <Tab
-            id="proposed-markets"
-            name={t('Proposed markets')}
-            settings={<MarketsSettings />}
-            menu={
-              <TradingAnchorButton
-                size="extra-small"
-                data-testid="propose-new-market"
-                href={externalLink}
-                target="_blank"
-              >
-                {t('Propose a new market')}
-              </TradingAnchorButton>
-            }
-          >
-            <ErrorBoundary feature="markets-proposed">
-              <Proposed />
-            </ErrorBoundary>
-          </Tab>
-          <Tab
-            id="closed-markets"
-            name={t('Closed markets')}
-            settings={<MarketsSettings />}
-          >
-            <ErrorBoundary feature="markets-closed">
-              <Closed />
-            </ErrorBoundary>
-          </Tab>
-        </Tabs>
+    <div className="pt-0.5 pb-3 px-1.5 xxl:px-[5.5rem]">
+      <div className="flex justify-between">
+        <div className="flex gap-2">
+          {Object.keys(marketTabs).map((key: string) => (
+            <button
+              key={key}
+              className={classNames(
+                'border border-default rounded-lg px-3 py-1.5 my-1 text-sm',
+                {
+                  'dark:bg-vega-cdark-800 bg-vega-clight-800':
+                    activeTab === marketTabs[key].id,
+                  'text-muted': activeTab !== marketTabs[key].id,
+                }
+              )}
+              id={marketTabs[key].id}
+              onClick={() => setActiveTab(marketTabs[key].id)}
+            >
+              {marketTabs[key].name}
+            </button>
+          ))}
+        </div>
+        <div className="flex">
+          {activeTab === 'proposed-markets' && (
+            <a
+              className="border border-default rounded-lg px-3 py-1.5 my-1 text-sm"
+              data-testid="propose-new-market"
+              href={externalLink}
+              target="_blank"
+              rel="noreferrer"
+            >
+              {t('Propose a new market')}
+            </a>
+          )}
+        </div>
+      </div>
+      <div className="h-full my-1 border rounded border-default">
+        {activeTab === 'open-markets' && (
```

Comment:
Perhaps this has already been discussed. But my expectation was that this would be a single table with ALL markets that would be programmatically filtered. 

Currently the columns are different between each state and we may still want that. If that is the case then this is probably suitable.

BUT its worth saying that in the current implementation, there won't be a way to see all markets with the same settlement asset (for example), regardless of its open/closed/proposed state.

For me, I think that kind of filtering is needed for this 'master' list. 

@JonRay15

Reply:
yes, the columns are different for open, closed and proposed markets. 
totally different tables because of that and that's why I made these tabs like this for now 

i think we can revisit this when we add more filters